teamtwo/naturescall/models.py

```diff
@@ -1,3 +1,8 @@
 from django.db import models
 
-# Create your models here.
+
```

Comment:
To document this class, comment:
`# Temporary class to hold fetched restroom/restaurant entries for their business id and rating.`